app/test_with_resources.bzl

```diff
@@ -0,0 +1,20 @@
+def test_with_resources(file):
```

Comment:
Nice idea to combine the genrule & the target! This is a great simplification.

Reply:
Thank you!